examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,195 @@
+)
+
+coil.sweep_around_axis(cs_axis="Z") # todo: what is the correct input?
+
+coil_terminal = m3d.modeler.create_rectangle(
+    csPlane="XZ", position=[70, 0, -11], dimension_list=[11, 110], name="Coil_terminal"
+)
+
+core = m3d.modeler.create_rectangle(
+    csPlane="XZ", position=[45, 0, -18], dimension_list=[7, 160], name="Core"
+)
+core.sweep_around_axis(cs_axis="Z")  # todo: what is the correct input?
+
+region = m3d.modeler.create_region(pad_percent=[20, 20, 500, 20, 20, 100])
+
+###############################################################################
+# Assign materials
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Coil --> copper, core --> ferrite, region --> vacuum
+
+m3d.assign_material(region.name, "vacuum")
+m3d.assign_material(coil.name, "copper")
+m3d.assign_material(core.name, "ferrite")
+
+###############################################################################
+# Assign excitation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# 10 turns, 50A
+
+m3d.assign_coil(["Coil_terminal"], conductor_number=10, name="Coil_terminal")
+
+m3d.assign_winding(
+    is_solid=False,
+    current_value=100,
+    name="Winding1",
+)
+
+m3d.add_winding_coils(windingname="Winding1", coil_names=["Coil_terminal"])
+
+###############################################################################
+# Assign mesh operations
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Not necessary in eddy current solver, but when assigned, they speed up the simulation
+
+m3d.mesh.assign_length_mesh(
+    ["Core"], maxlength=15, meshop_name="Inside_Core", maxel=None
+)
+m3d.mesh.assign_length_mesh(
+    ["Coil"], maxlength=30, meshop_name="Inside_Coil", maxel=None
+)
+
+###############################################################################
+# Set conductivity temperature coefficient
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Conductivity is now function of temperature
+
+Cu = m3d.materials["copper"]
+cu_resistivity_temp_coefficient = 0.00393
+Cu.conductivity.add_thermal_modifier_free_form("1.0/(1.0+{}*(Temp-20))".format(cu_resistivity_temp_coefficient))
+
+###############################################################################
+# Set object temperature and enable feedback
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Fro 2-way coupling
+
+m3d.modeler.set_objects_temperature(["Coil"])
+
+###############################################################################
+# Assign matrix
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# RL calculation
+
+m3d.assign_matrix(["Winding1"], matrix_name="Matrix1")
+
+###############################################################################
+# Create and analyze simulation setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Simulation frequency 150kHz
+
+setup = m3d.create_setup(setupname="Setup1")
+setup.props["Frequency"] = "150kHz"
+m3d.analyze_setup("Setup1")
+
+# todo: how to get the resistance?
+#resistance = m3d.post.get_solution_data("Matrix1.R(Winding1,Winding1)").full_matrix_mag_phase[0]
```

Comment:
@IreneWoyna if you are looking for resistance values you could create the report first:
`report = m2d.post.create_report(
    expressions="Matrix1.R(Winding1,Winding1)",
    primary_sweep_variable=your_primary_sweep,
    report_category=your_report_category,
    plot_type=your_plot_type,
    variations=variations,
    plotname=plot_name,
)
solution= report.get_solution_data()
resistance = solution.data_magnitude()`